src/_guides/language/effective-dart/design.md

```diff
@@ -707,7 +707,13 @@ var buffer = new StringBuffer()
 
 ## Type annotations
 
-In Dart, adding static types to your variables is optional.
+In Dart 1.x, adding static types to your variables is optional.
+
```

Comment:
It seems like we might want to explicitly mark certain sections as likely to change in 2.0. Maybe we could even point to related issues on github?

/cc @munificent